lisa/features/sriov.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any
+
+from lisa.feature import Feature
+from lisa.platform_ import Platform
+from lisa.sut_orchestrator.azure.common import get_network_client
+from lisa.util import SkippedException
+from lisa.util.logger import get_logger
+
+FEATURE_NAME_SRIOV = "Sriov"
+
+
+class Sriov(Feature):
+    def __init__(self, node: Any, platform: Any) -> None:
+        super().__init__(node, platform)
+        self._log = get_logger("feature", self.name(), self._node.log)
+
+    @classmethod
+    def name(cls) -> str:
+        return FEATURE_NAME_SRIOV
+
+    def _is_supported(self) -> bool:
+        raise NotImplementedError()
+
+    def check_support(self) -> None:
+        if not self._is_supported():
+            raise SkippedException("Sriov is not supported.")
+
+    def enable(self) -> bool:
+        return self.disable_enable_sriov(self._platform, True)
+
+    def disable(self) -> bool:
+        return self.disable_enable_sriov(self._platform, False)
+
+    def disable_enable_sriov(self, platform: Platform, enable: bool) -> bool:
```

Comment:
1. if it's called inside of sriov only, rename it like `_switch`.
2. The content is Azure specific, so the implementation should be in Azure package, like Gpu.